package.json

```
@@ -7,7 +7,12 @@
     "build": "lerna run build",
     "test": "lerna run test",
     "package": "lerna run package && tools/collect-dist.sh",
-    "release": "tools/release.sh"
+    "release-github": "tools/release-github.sh",
+    "release-maven": "jsii-release-maven",
+    "release-npm": "jsii-release-npm",
+    "release-nuget": "jsii-release-nuget",
+    "release-pypi": "jsii-release-pypi",
+    "release": "echo 'You must specify a package manager to release this on. Try \"yarn release-npm\"'"
```

Not sure this is needed:

```suggestion
    "release-pypi": "jsii-release-pypi"
```